### DISCUSSION TITLE
TicketBai 单据开错了客户资料不对,商品价格不对, 又不想打退单直接编辑单据功能,该怎么做?

### DISCUSSION THREAD

**Discussion by m916423330** [densen2014/docs#13] [Q&A] [answered]
同题目

**Comment by densen2014** [comment, marked as answer]
https://github.com/densen2014/docs/wiki/TICKETBAI-%E7%AE%97%E8%B4%A6%E8%AF%B4%E6%98%8E%E4%B9%A6#4%E5%8D%95%E6%8D%AE%E5%BC%80%E9%94%99%E4%BA%86%E5%AE%A2%E6%88%B7%E8%B5%84%E6%96%99%E4%B8%8D%E5%AF%B9%E5%95%86%E5%93%81%E4%BB%B7%E6%A0%BC%E4%B8%8D%E5%AF%B9-%E5%8F%88%E4%B8%8D%E6%83%B3%E6%89%93%E9%80%80%E5%8D%95%E7%9B%B4%E6%8E%A5%E7%BC%96%E8%BE%91%E5%8D%95%E6%8D%AE%E5%8A%9F%E8%83%BD
